[SOLVED] Error in building:  Can't copy 'po/*.mo' : doesn't exist or not regular file

** For how to run the latest Guake in your computer, please refer to [Install from source]
** (https://guake.readthedocs.io/en/latest/user/installing.html#install-from-source).<
I did..
I ran script bootstrap to install all neccesaries.
Ran 'make', then 'sudo make install' and got the following error:

`
error: can't copy 'po/*.mo': doesn't exist or not a regular file
make: *** [Makefile:93: install-guake] Fout 1
`

I just got me a new system with Mint 20 (previous was 19) and i wanted the latest quake back. But the installation is not going smooth... i have no clue how to fix this. Appreciating some help.

Thanks

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -5,3 +5,4 @@ include guake *
 prune guake/tests
 exclude guake/paths.py.in
 include guake/paths.py
+include po/*.mo

--- a/setup.cfg
+++ b/setup.cfg
@@ -31,9 +31,6 @@ install_requires =
     importlib_metadata; python_version < '3.8'
     typing; python_version < '3.5'
 
-[options.data_files]
-share/guake/po = po/*.mo
-
 [options.entry_points]
 console_scripts =
     guake = guake.main:exec_main

